### DISCUSSION TITLE
[Help]: Change `scrollSpeed` for the Fade plugin with Autoplay

### DISCUSSION THREAD

**Discussion by pras-wjy** [davidjerleke/embla-carousel#1029] [Help] [answered]
### Which variants of Embla Carousel are relevant to this feature request?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [X] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [x] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

I hope there is an option to control the autoplay scroll speed.

my use case is using autoplay plugin with fade plugin, and i want the fade transition to be slower/smoother:
```tsx
const [emblaRef, emblaApi] = useEmblaCarousel(
  {loop: false, watchDrag: false},
  [
    Fade(),
    AutoPlay({
        delay: 4000,
        scrollSpeed: 1000, // controlling fade duration
    })
  ] 
)
```

### Additional Context

Additional details here...

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by davidjerleke** [comment, marked as answer]
Hi @prs-wjy,

Why don't you use the duration option for the carousel like so:

```tsx
const [emblaRef, emblaApi] = useEmblaCarousel(
  {
    loop: false, 
    watchDrag: false,
    duration: 30 // controlling fade duration
  },
  [
    Fade(),
    AutoPlay({ delay: 4000 })
  ]
)
```

**Reply by pras-wjy** [reply]
it works! Thanks!

i didn't expect it's in the base options, I thought because the scroll is done by `autoplay`, the options gonna be in the `autoplay` plugin options.
